pkg/inject/sidecar.go

```diff
@@ -309,6 +310,31 @@ func container(jaeger *v1.Jaeger, dep *appsv1.Deployment, agentIdx int) corev1.C
 		SecurityContext: jaeger.Spec.Agent.SidecarSecurityContext,
 		VolumeMounts:    volumesAndMountsSpec.VolumeMounts,
 	}
+
+	if isContainerPortAvailable(adminPort, dep) {
+		containerDefinition.LivenessProbe = &corev1.Probe{
+			ProbeHandler: corev1.ProbeHandler{
+				HTTPGet: &corev1.HTTPGetAction{
+					Path: "/",
+					Port: intstr.FromInt(int(adminPort)),
+				},
+			},
+			InitialDelaySeconds: 5,
```

Comment:
Why this values? where it comes from?

Reply:
@rubenvp8510, all those values were carried over from the previous PR, so I'm not completely sure of why they were originally set that way. I tested on a few clusters and they seemed to be fine, so I didn't really see a need to mess with them. But if you have any suggestions, I'd be happy to explore further.